src/main/java/org/broadinstitute/consent/http/db/DataSetDAO.java

```diff
@@ -99,6 +99,13 @@
             "where d.dataSetId = :dataSetId order by d.dataSetId, k.displayOrder")
     Set<DataSetDTO> findDataSetWithPropertiesByDataSetId(@Bind("dataSetId") Integer dataSetId);
 
+  @UseRowMapper(DataSetPropertiesMapper.class)
+  @SqlQuery("select d.*, k.key, dp.propertyValue, ca.consentId, c.dac_id, c.translatedUseRestriction " +
+      "from dataset d left outer join datasetproperty dp on dp.dataSetId = d.dataSetId left outer join dictionary k on k.keyId = dp.propertyKey " +
+      "left outer join consentassociations ca on ca.dataSetId = d.dataSetId left outer join consents c on c.consentId = ca.consentId " +
+      "where d.dataSetId = :dataSetId order by d.dataSetId, k.displayOrder")
```

Comment:
Couple bits of feedback here.
* I like upper cased sql operators - it's easier to read
* Postgres doesn't require mixed case column names - they are all lowercased
* I also like white space to break up the sql to make it more legible
This should look something like this:
```
SELECT d.*, k.key, dp.propertyvalue, ca.consentid, c.dac_id, c.translateduserestriction
FROM dataset d 
LEFT OUTER JOIN datasetproperty dp on dp.datasetid = d.datasetid 
LEFT OUTER JOIN dictionary k on k.keyid = dp.propertykey
LEFT OUTER JOIN consentassociations ca on ca.datasetid = d.datasetid 
LEFT OUTER JOIN consents c on c.consentid = ca.consentid 
WHERE d.datasetid = :datasetId 
ORDER BY d.datasetid, k.displayorder
```